tests/integration/helpers.py

```diff
@@ -3,34 +3,46 @@
 
 """Helper functions and classes for integration test."""
 
+import json
+import logging
+import textwrap
+from pathlib import Path
+
 import requests
 from juju.application import Application
 from juju.model import Model
 from juju.unit import Unit
 
 from state import CACHE_CONFIG_INTEGRATION_NAME
 
+logger = logging.getLogger(__name__)
+
+TEST_SERVER_PATH = Path("tests/integration/scripts/test_server.py")
+
+HOSTNAME_CONFIG_NAME = "hostname"
+PATH_CONFIG_NAME = "path"
+BACKENDS_CONFIG_NAME = "backends"
+BACKENDS_PATH_CONFIG_NAME = "backends-path"
+PROTOCOL_CONFIG_NAME = "protocol"
+FAIL_TIMEOUT_CONFIG_NAME = "fail-timeout"
+PROXY_CACHE_VALID_CONFIG_NAME = "proxy-cache-valid"
+
 
 class CacheTester:
     """Test content cache.
 
     Attributes:
-        TEST_CONFIG: The cache configuration for testing.
-        EMPTY_CONFIG: The empty cache configuration.
+        BASE_CONFIG: The base cache configuration.
     """
 
-    TEST_CONFIG = {
-        "hostname": "test.local",
-        "path": "/",
-        "backends": "20.27.177.113",  # A IP to github.com
-        "protocol": "http",
-    }
-
-    EMPTY_CONFIG = {
-        "hostname": "",
-        "path": "/",
-        "backends": "",
-        "protocol": "https",
+    BASE_CONFIG = {
+        HOSTNAME_CONFIG_NAME: "",
+        PATH_CONFIG_NAME: "/",
+        BACKENDS_CONFIG_NAME: "",
+        BACKENDS_PATH_CONFIG_NAME: "/",
+        PROTOCOL_CONFIG_NAME: "https",
+        FAIL_TIMEOUT_CONFIG_NAME: "30s",
+        PROXY_CACHE_VALID_CONFIG_NAME: "[]",
```

Comment:
Nitpick: these are named `*_FIELD_NAME` in the code. Maybe it would be better to name them same here.

Reply:
It issue is the integration data (`*_FIELD_NAME`) uses underscores `_` as space.
And the configuration name (`*_CONFIG_NAME`) uses dash `-` as space.
E.g., `fail_timeout` and `fail-timeout`.